Clone-a-specific-branch.md

```diff
@@ -0,0 +1,7 @@
+<h1>Clone a specific branch</h1>
+There are many cases where we want to clone a particular branch on oue loacal machine. For that we need to follow these methods:
+
+git clone -b <branch_name> <remote_repo>
+<p>This will help to clone specific branch. Please note here public key setup is required. If you don't have public key setup then also you can clone by</p>
+<p>Example, with OpenCV 2.4 branch:</p>
+git clone -b 2.4 --single-branch https://github.com/Itseez/opencv.git opencv-2.4
```

Comment:
@aj07 This is what he is talking about [Formatting Code}(https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/code-formatting)

Also please use Markdown instead of HTML tags. For headers you can use # for h1, and if you want h3 for example you can use # three times in a row.

You don't need to declare paragraphs. For Bold surround the words with *